source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -558,65 +383,45 @@ def get(self, region=None, lon=None, lat=None, uri=None):
             if args.get('max_duration') is None:
                 args['max_duration'] = app.config['ISOCHRONE_DEFAULT_VALUE']
 
-        def _set_specific_params(mod):
-            if args.get('max_duration') is None:
-                args['max_duration'] = mod.max_duration
-            if args.get('_walking_transfer_penalty') is None:
-                args['_walking_transfer_penalty'] = mod.walking_transfer_penalty
-            if args.get('_night_bus_filter_base_factor') is None:
-                args['_night_bus_filter_base_factor'] = mod.night_bus_filter_base_factor
-            if args.get('_night_bus_filter_max_factor') is None:
-                args['_night_bus_filter_max_factor'] = mod.night_bus_filter_max_factor
-            if args.get('_max_additional_connections') is None:
-                args['_max_additional_connections'] = mod.max_additional_connections
-
         if region:
-            _set_specific_params(i_manager.instances[region])
+            if args.get('max_duration') is None:
+                args['max_duration'] = i_manager.instances[region].max_duration
         else:
-            _set_specific_params(default_values)
+            if args.get('max_duration') is None:
+                args['max_duration'] = default_values.max_duration
+
 
         if not (args['destination'] or args['origin']):
             abort(400, message="you should at least provide either a 'from' or a 'to' argument")
 
         if args['debug']:
             g.debug = True
 
-        #we transform the origin/destination url to add information
-        if args['origin']:
-            args['origin'] = transform_id(args['origin'])
-        if args['destination']:
-            args['destination'] = transform_id(args['destination'])
-
-        if not args['datetime']:
-            args['datetime'] = datetime.now()
-            args['datetime'] = args['datetime'].replace(hour=13, minute=37)
-
         if not region:
-            #TODO how to handle lon/lat ? don't we have to override args['origin'] ?
+            # TODO how to handle lon/lat ? don't we have to override args['origin'] ?
```

Comment:
I think it would be better to encapsulate this in a compute_regions method and use it in journeys and isochrones